### DISCUSSION TITLE
Welcome to Live Sass Compiler Discussions!

### DISCUSSION THREAD

**Discussion by glenn2223** [glenn2223/vscode-live-sass-compiler#180] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community/extension.

**Comment by evilgeniuscreative** [comment]
1) Why are so many of the settings forced to set in json and not in the app? 
2) How do you set the output folder instead of the same folder that the scss files are in?
3) This extension needs a happy path with clear options. It's confusing for those of us who aren't VS Code nerds.

**Reply by glenn2223** [reply]
Hey @evilgeniuscreative, thanks for voicing in on our little community 😁

> 1. Why are so many of the settings forced to set in json and not in the app?

VS code only allows: numbers, strings, arrays of the previous two and checkboxes to be editable from the GUI. You'll have to reach out to them or submit a PR for them to extend this. Oh, and you'll find that the "in the app" GUI is just a different view of the JSON file - so you're looking at the same thing, just albeit in a prettier view. I also revived the dead extension so am bound to alot if the original settings/formats to make migration easier. 

> 2\. How do you set the output folder instead of the same folder that the scss files are in?

You can check the settings doc or refer to the [YouTube video](https://youtu.be/6Wo3mYBLNyA) to hear and see what they do (it's my first one, so please bear with me as it's far from perfect). In this particular case you will want to use the `formats` setting and set either `savePath` or `savePathReplacementPairs` or a combination of both.

> 3\. This extension needs a happy path with clear options. It's confusing for those of us who aren't VS Code nerds.

Providing you need the files saved in the same location and you follow standard SASS naming principles, you should find the extension is pretty much install and go. Also, I'm afraid VS Code is so stripped back you need to be a VS Code nerd to do most things. Extensions like mine are bound to the platform defined way and do our best with online docs to support

---

I hope this helps a little and is informative. I'm also open to non-breaking PRs/feature requests, if there's anything you want to submit

**Comment by evilgeniuscreative** [comment]
Thank you for your fast reply. Is there anything other than a 45 minute
video to get basic instructions on?

Thanks

Ian

**Reply by glenn2223** [reply]
Yeah, there's the docs that break it down - if you'd rather read about it. 

The video is a deep dive in to all the settings and commands of the extension. If you're just after changing the output location then take a look at the 5 minute section, [starting here](https://www.youtube.com/watch?v=6Wo3mYBLNyA&t=487s), about the `formats` setting.

The whole video is timestamped to try and help identify the bits you probably care about. Starting with the intro and a preview; then leading onto each setting (and sub-components in the case of the formats setting) and command

**Comment by evilgeniuscreative** [comment]
Thanks
